semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/TextDocumentOps.scala

```diff
@@ -605,6 +604,10 @@ trait TextDocumentOps { self: SemanticdbOps =>
             }
 
             gtree match {
+              case apply: g.Apply
+                  if gtree.pos != null && gtree.pos.isRange && apply.fun.symbol.isCaseApplyOrUnapply && apply.tpe.typeSymbol.isCaseClass =>
```

Comment:
This is too much on one line

Reply:
Changed.

Reply:
You probably forgot to push :sweat_smile: